### DISCUSSION TITLE
pattern-match/intro

### DISCUSSION THREAD

**Discussion by giscus** [sunface/rust-by-practice#328] [Book Comments]
# pattern-match/intro

Learning Rust By Practice, narrowing the gap between beginner and skilled-dev with challenging examples, exercises and projects.

https://zh.practice.rs/pattern-match/intro.html

**Comment by somecodework** [comment]
done

**Comment by DylanJinx** [comment]
i am coming!

**Reply by GMatStarcraft** [reply]
comin for what
